data/style.scss

```diff
@@ -9,13 +9,37 @@ $themes: squirrel, archlinux, monokai, zenburn;
 // in a selector list.
 @function body_not_themed() {
     $s: unquote("body");
+
     @each $t in $themes {
         $s: selector.append($s, unquote(":not(.theme_#{$t})"));
     }
+
     @return $s;
 }
 
+// make QR code expand and fill page
+// --------------------------------------------------
```

Comment:
I believe these indicators are unnecessary. We use them in no other location.

Reply:
Removed.